demos/redpanda/ockam-setup

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+NAME="kafkabroker"
+
+ockam reset
+ockam enroll
+
+OCKAM_BROKER_TOKEN=$(ockam project ticket --attribute component=kafkabroker)
+
+ockam identity create "$NAME"_identity
+ockam project enroll $OCKAM_BROKER_TOKEN --identity "$NAME"_identity
+ockam node create $NAME --identity "$NAME"_identity
```

Comment:
We have to TCP listening address here because or note the address that is randomly chosen because producer and consumer need to know a route to the outlet.